web/quiz_views.py

```
@@ -149,50 +223,77 @@ def add_question(request, quiz_id):
+                true_option = QuizOption(question=question, text="True", is_correct=true_correct, order=0)
+                true_option.save()
 
-        if form_valid and formset_valid:
-            try:
+                # Create or update the "False" option
+                false_option = QuizOption(question=question, text="False", is_correct=not true_correct, order=1)
+                false_option.save()
+
+            messages.success(request, "Question added successfully.")
+            return redirect("quiz_detail", quiz_id=quiz.id)
+        elif question_type == "multiple":
```

Ruff (0.8.2)

Unnecessary `elif` after `return` statement

Remove unnecessary `elif`

(RET505)